test/test_list.rb

```diff
@@ -882,3 +882,52 @@ def test_insert_at
     assert_equal 3, new.pos
   end
 end
+
+class NoUpdateForCollectionClassesTest < ActsAsListTestCase
+  def setup
+    setup_db
+
+    @mixin_1, @mixin_2 = (1..2).map { |counter| ListMixin.create!(pos: counter) }
+    @sub_mixin_1, @sub_mixin_2 = (1..2).map { |counter| ListMixinSub1.create!(pos: counter) }
+  end
+
+  def test_no_update_for_single_class
+    ListMixin.acts_as_list_no_update { @mixin_1.update_attributes(pos: 2) }
+    assert_equal @mixin_1.pos, 2
+    assert_equal @mixin_2.pos, 2
+  end
+
+  def test_no_update_for_different_class
+    ListMixin.acts_as_list_no_update([ListMixinSub1]) do
+      udpate_mixin_and_sub_mixin
+    end
+
+    assert_equal @mixin_1.pos, 2
+    assert_equal @mixin_2.pos, 2
+    assert_equal @sub_mixin_1.pos, 2
+    assert_equal @sub_mixin_2.pos, 2
+  end
+
```

Comment:
I think you need a test here that tests that callbacks are still performed on a non-specified `acts_as_list` class when called within the block (as per the `README` example update.)